gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -3020,4 +3020,7 @@ public SecurityAdvisor addSecurityAdvisor() {
 	public void removeSecurityAdvisor(final SecurityAdvisor advisor) {
 		this.securityService.popAdvisor(advisor);
 	}
+	public boolean getIsCalculatedGrade() {
```

Comment:
this is a confusing method name. I would recommend showCalculatedGrade and getShowCalculatedGrade

Reply:
Of course.